nautobot/extras/utils.py

```diff
@@ -80,6 +82,42 @@ def get_choices(self):
         return [(f"{ct.app_label}.{ct.model}", ct.pk) for ct in ContentType.objects.filter(self.get_query())]
 
 
+@deconstructible
+class ModelSubclassesQuery:
+    """
+    Helper class to get ContentType models that are subclasses of self.klass
+    """
+
+    def __init__(self, klass):
+        self.klass = import_object(klass)
```

Comment:
I'm a bit surprised that it works to immediately import klass at declaration time, as far as circular imports and such go. I would have thought you'd need to defer the import_object call until `list_subclasses()` gets evaluated, but if it works as is, great!